pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go

```diff
@@ -202,6 +202,10 @@ type KubeletConfiguration struct {
 	CgroupDriver string `json:"cgroupDriver,omitempty"`
 	// containerRuntime is the container runtime to use.
 	ContainerRuntime string `json:"containerRuntime"`
+	// CPUManagerPolicy is the name of the policy to use.
+	CPUManagerPolicy string `json:"cpuManagerPolicy"`
+	// CPU Manager reconciliation period.
+	CPUManagerReconcileFrequency metav1.Duration `json:"cpuManagerReconcileFrequency"`
```

Comment:
in future if we have a dynamic policy, i assume this is the heartbeat.

Reply:
Frequency is not correct - we would usually call this `Interval` (I'm fairly certain this is in api-conventions).  Frequency would be an inverse duration.

Reply:
@smarterclayton -- this is consistent with SyncFrequency, FileCheckFrequency, HTTPCheckFrequency, and NodeStatusUpdateFrequency in kubelet for this type.

https://github.com/ConnorDoyle/kubernetes/blob/ba3bf1f12be7024bd7c50f9ed14e3a58cf15a341/pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go#L61

I didn't see a strong statement here:
https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#naming-conventions

Reply:
we are not super consistent either as we sometimes use "*Period" (volumeStatsAggPeriod)

https://github.com/ConnorDoyle/kubernetes/blob/ba3bf1f12be7024bd7c50f9ed14e3a58cf15a341/pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go#L180

Reply:
@smarterclayton totally agree -- initially it was --CPU-manager-reconcile-period but I changed it to be more consistent. Am OK with whatever option name the approvers like. Just let me know what you would like it to be called and I will rename post-haste.

Reply:
The mentioned kubelet flags are some of the oldest in the codebase and predate our guidelines for this.  We would not accept them now (and kubelet is the only offender in this space).

`Period` is the correct value for a field of time.Duration that covers the interval between two attempts.  There are multiple uses in the codebase across more than one area (much newer than the original frequency) and it's also part of ObjectMeta (TerminationGracePeriod).  I think it just needs to get into conventions, an oversight.

Don't have to change the other flags (API compat) but future flags in this package should all be `Period`.

Reply:
Clarify - fix manager reconcile period here and note to other reviewers as well.  I'll try to remember to update api-conventions.